index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Mobile Portfolio</title>
+</head>
+<body>
+    <main>
+        <header class="header"> 
+            <a href="#">Bwalya Nonde</a>
+            <nav class="nav-menu">
+                <img src="images/union.png" alt="">
+            </nav> 
+        </header>
+        <div class="intro-photo"></div>
+        <section class="introduction">
+            <div class="intro-details">
+                <h1>Bwalya Nonde</h1>
+                <h2>Software Developer</h2>
+                <p class="intro-paragraph">
+                    I can help you build a product , feature or website Look through some of 
+                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                
+            </div>
```

Comment:
Please add all the social icons which can be seen in the design. The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).
![icons](https://user-images.githubusercontent.com/35504149/164799061-1a55b24b-54d7-4ead-bd47-21ffbb8e93ed.png)